docs/tasks/tls/managing-tls-in-a-cluster.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Managing TLS Certificates in a Cluster
+assignees:
+- mikedanese
+- beacham
+- liggit
+---
+
+* TOC
+{:toc}
+
+## Overview
+
+Every Kubernetes has a cluster root Certificate Authority (CA). The CA is
+generally used by cluster components to validate the API server's certificate,
+by the API server to validate kubelet client certificates, etc. To support
+this, the CA certificate bundle is distributed to every node in the cluster and
+is distributed as a secret attached to default service accounts.  Optionally,
+your workloads can use this CA to establish trust. Your application can request
+a certificate signing using the `certificates.k8s.io` API using a protocol
+that is similar to the [ACME draft](https://letsencrypt.github.io/acme-spec/).
+
+## Trusting TLS in a Cluster
+
+Trusting the cluster root CA from an application running as a pod usually
+requires some extra application configuration. You will need to add the CA
+certificate bundle to the list of CA certificates that the TLS client or server
+trusts. For example, you would do this with a golang TLS config by parsing the
+certificate chain and adding the parsed certificates to the `Certificates` field
+in the [`tls.Config`](https://godoc.org/crypto/tls#Config) struct.
+
+The CA certificate bundle is automatically mounted into pods using the default
+service account at the path `/var/run/secrets/kubernetes.io/serviceaccount/ca.crt`.
+If you are not using the default service account, ask a cluster administrator to
+build a configmap containing the certificate bundle that you have access to use.
+
+## Requesting a Certificate
+
+Here we will be demonstrating how to create a TLS certificate for a Kubernetes
```

Comment:
Avoid "we" constructs.

"The following section demonstrates how to create a TLS certificate..."